tools/user_migration/migrate_node.py

```diff
@@ -451,34 +459,52 @@ def spanner_transact_bso(transaction, data, fxa_kid, fxa_uid, args):
         cursor.close()
     return count
 
+def get_users(args, databases, fxa, bso_num):
+    users = []
+    cursor = databases['mysql'].cursor()
+    if args.user:
+        users = args.user
+    else:
+        try:
+            sql = """select distinct userid from bso{} order by userid""".format(bso_num)
+            if args.user_range:
+                (offset, limit) = args.user_range.split(':')
+                sql = "{} limit {} offset {}".format(sql, limit, offset)
+            cursor.execute(sql)
+            for (user,) in cursor:
+                try:
+                    (fxa_kid, fxa_uid) = fxa.get(user)
+                    users.append((user, fxa_kid, fxa_uid))
+                except TypeError:
+                    logging.error("⚠️User not found in tokenserver data: {} ".format(
+                        user
+                    ))
+            users.sort(key=lambda tup: tup[2])
+        except Exception as ex:
+            import pdb; pdb.set_trace()
```

Comment:
✂️ ?

Reply:
well, it's in an exception, and the script is meant to be hand-run. I'd rather it dump to an examinable state at that point.